grepos/app/src/main/java/net/boondockradio/grepos/MainActivity.java

```diff
@@ -22,51 +23,78 @@
 
 public class MainActivity extends AppCompatActivity {
 
-    private final String TAG = "grepos";
+    private final String TAG = MainActivity.class.getSimpleName();
+    private final int LOAD_MORE_THRESHOLD = 1;
+    private final String PER_PAGE = "50";
 
     private List<Repository> mRepositoryItems;
-    private RecyclerView mRecyclerView;
-    private RecyclerView.Adapter mAdapter;
+    private RepositoryAdapter mAdapter;
+    private ProgressBar mProgress;
+
+    private int mPage = 0;
+
+    private int total = 0;
+    private int last = 0;
+    private boolean isLoading = false;
```

Comment:
if we are strict we should use hungarian notation ie `m`

``` java
private int mTotal = 0;
private int mLast = 0;
private boolean isLoading = false;
```

we hope to remove hungarian notation soon